modules/ROOT/pages/partner-manager-architecture.adoc

```diff
@@ -1,2 +1,63 @@
 = Anypoint Partner Manager Architecture
 
+With Anypoint Partner Manager, you can deploy B2B Integration message flows with your partners, either to your own hosted Mule runtimes or to MuleSoft hosted CloudHub iPaaS platform.
+
+The functional features and user experience in configuring B2B Integrations with Partner Manager are deployment-target agnostic; however, there are differences in the underlying architecture when implementing the functionality in the runtime.
```

Comment:
Does "the runtime" refer to the Mule runtime engine? 
Do we mean "in Mule" or "in the Mule instance"?
GLOBALs

Reply:
In some cases, when they say "in the runtime", that is the phrasing used in other parts of the doc. For instances where it says "Mule runtime" that can be changed to "Mule instance"